ui/pages/providers/index.js

```diff
@@ -24,7 +24,7 @@ export default function Providers() {
         <h1 className='py-1 font-display text-xl font-medium'>Providers</h1>
         <Link href='/providers/add'>
           <a className='inline-flex items-center rounded-md border border-transparent bg-black px-4 py-2 text-xs font-medium text-white shadow-sm hover:cursor-pointer hover:bg-gray-800'>
-            Connect provider
+            Connect Provider
```

Comment:
then maybe we also need to change the `Connect cluster` to `Connect Cluster` on the infrastructure page too

Reply:
Lowercase was the convention so I changed this back to `Connect provider`